tox.ini

```diff
@@ -1,16 +1,18 @@
 [coverage:report]
 skip_covered = False
 show_missing = True
-# TODO: SNOW-1453027 remove omit
+# TODO: SNOW-1453027 remove omit for config and core
 omit = */src/snowflake/snowpark/modin/config/*
        */src/snowflake/snowpark/modin/core/*
+       */src/snowflake/snowpark/modin/plugin/docstrings/*
```

Comment:
@sfc-gh-helmeleegy we can just skip it here.

Reply:
Got it.